content/1.7/back-office/shop-parameters/search/search-listing.md

```diff
@@ -0,0 +1,108 @@
+
+1. Aliases
+
+Name of the aliases
+
+2.  Search
+
+Result of the alias 
+
+3. Status
+
+Checkmark if the alias is enabled or disabled
+
+5. Actions
+
+- **Edit.** This is the main action and it can be performed by clicking on the line or on the button. 
+👉 See the [specs of the add/edit alias](./add-edit-alias.md) 
+
+- **Delete.** The alias can be deleted by clicking on this button. After clicking on "Delete", a modal with the title "Delete selection" is opened asking to confirm or cancel the  action: "Are you sure you want to delete the selected item(s)?"
+  The action can be canceled by clicking on the cross or on the cancel button.
+  After clicking on "Delete" button, the tag is deleted and a successful message is displayed "Successful deletion."
+  
+ ## Bulk actions
+
+Clicking on the _Bulk actions_ button allows users to perform bulk actions for selected feature values (using the checkboxes). A user can:
+
+- Select all/Unselect all: this button will select/unselect all the checkboxes at the beginning of each row of the listing.
+- Enable selection. Enables the selected alias, changes the status to green check mark. If no alias is selected when pressing "enable selection" the message is shown: "You must select at least one item to perform a bulk action." If an alias is selected, and "enable" is pressed, the message is shown: "The status has been successfully updated." and check mark is changed to a green one.
+- Disable selection. Disables the selected alias, changes the status to red cross mark. If no alias is selected when pressing "disable selection" the message is shown: "You must select at least one item to perform a bulk action." If an alias is selected, and "disable selection" is pressed, the message is shown: "The status has been successfully updated." and check mark is changed to a red cross.
+- Delete selected: This button will delete the selected alias. 
+After clicking on "Delete selected", a modal with the title "Delete selection" is opened asking to confirm or cancel the action: "Are you sure you want to delete the selected item(s)?" (See issue [#14462](https://github.com/PrestaShop/PrestaShop/issues/14462)). The action can be canceled by clicking on the cross or on the cancel button.
+After clicking on "Delete" button, the selected feature values are deleted, a successful message is displayed "The selection has been successfully deleted."
+
+ ## Search
+ 
+The search button is disabled as long as there are less than 2 alias  in the list. A search can be done by aliases, search, and status.
+It's possible to sort the list by aliases, search in an ascending or descending way.
+After clicking on "search", the "search" button is disabled and a "reset" button is displayed.
+If there is no results for the search done, the following message is displayed: "No records found".
+When clicking on "reset", the search is removed and all the results are displayed.
+
+ ## Indexing
```

Comment:
```suggestion
# Indexing
```

Reply:
I let you improve this part to make it more easier to read (add line breaks)
<img width="1068" alt="Capture d’écran 2021-11-26 à 15 01 50" src="https://user-images.githubusercontent.com/13449658/143592283-27ef7d42-14bc-415d-8feb-ade2758a54ce.png">